ls1intum/artemis-android
# `Feature`: Add navigation animations

### Problem Description
Currently there are no animations when navigating the app. There should be smooth animations when navigating from one screen to another.

### Changes
 
- Added navigation animations to all screens
- Adapted Server instance selection to show as BottomSheet instead of own screen

I have tested that adding the animations does not mess with the different layouts on both tablets and foldables.

### Steps for testing
Navigate through all the app's screens and see that there are now animations when navigating
